### PR TITLE
Feature/DOC-14

### PR DESCRIPTION
**Pull request by shikhanansi**
https://ephocks.atlassian.net/browse/DOC-14. Added a note in 'paste_word_valid_elements'. Please review.

Regards,
Shikha

### CODE REVIEW

**Comment by techtangents** [comment]
No issues from me. I just need someone else to review for technical accuracy.

File: plugins/paste.md [hunk 1]
```diff
@@ -168,6 +168,8 @@ tinymce.init({
 
 This option enables you to configure the `valid_elements` specific to MS Office. Word produces a lot of junk HTML, so when users paste things from Word we do extra restrictive filtering on it to remove as much of this as possible. This option enables you to specify which elements and attributes you want to include when Word contents are intercepted.
 
+>**Note:** To access this feature, you need to set the value of [paste_enable_default_filters]({{site.baseurl}}./#paste_enable_default_filters) to `"false"` in your configuration.
```
Comment (TheSpyder) [inline plugins/paste.md]: Would just `[paste_enable_default_filters](#paste_enable_default_filters)` work since it's in this page?

**Comment by shikhanansi** [comment]
Hi Andy, 
Thank you for your review. Adding a link to the right section provides both reference and ease to the user to perform the exact step. It's also good documentation practice to not make the user "look-for" information and provide it all in one place, whenever possible.
Regards,
Shikha

**Comment by TheSpyder** [comment]
@shikhanansi I understand that, I wasn't requesting the link be removed, I was suggesting a more simple link to achieve the same result (since the section is on the same page).

**Pull request merged**